src/nx/nxVirtualKeyboard.c

```diff
@@ -89,8 +89,12 @@ uint8_t inputVirtualKeyboardText(const char *title, uint16_t maxLength, char *ou
             strncpy(outText, tmpoutstr, maxLength);
 
             success = 1;
+        } else {
+            spLogInfo("Failed to show software keyboard"); 
         }
         swkbdClose(&kbd);
+    } else {
+        spLogInfo("Failed to create software keyboard"); 
```

Comment:
Sorry about this nit 😂 but I'm loyal to the K&R style for curly braces 😅 
```suggestion
        }
        else
        {
            spLogInfo("Failed to show software keyboard"); 
        }
        swkbdClose(&kbd);
    }
    else
    {
        spLogInfo("Failed to create software keyboard"); 
```

Reply:
Oh no, my bad, I didn't pay too much attention to the code style 😓